cfgov/jinja2/v1/_layouts/base-common.html

```diff
@@ -19,7 +19,6 @@
 -->
 {% endif %}
 
-<!--[if lt IE 7]>      <html lang="es" class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
 <!--[if IE 7]>         <html lang="es" class="no-js lt-ie9 lt-ie8"> <![endif]-->
```

Comment:
While you're here, please fix the `lang` attribute here to have a value of `en`.